projects/common-components/src/lib/pipes/sort/sort.pipe.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { SortPipe } from "./sort.pipe";
+import { Result, Talk } from "@jsmu/core";
+import { TestBed } from "@angular/core/testing";
+
+describe("SortPipe", () => {
+	let pipe: SortPipe;
+	let talks: Talk[];
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			providers: [SortPipe]
+		});
+
+		pipe = TestBed.inject(SortPipe);
+		talks = [
+			{
+				lessonId: "",
+				menteeId: "",
+				expertId: "",
+				inProgress: true,
+				result: Result.comment,
+				resultDate: new Date(new Date().getTime()),
+				submitDate: new Date()
+			},
+			{
+				lessonId: "",
+				menteeId: "",
+				expertId: "",
+				inProgress: true,
+				result: Result.comment,
+				resultDate: new Date(new Date().getTime() - 100000000),
+				submitDate: new Date()
+			}
+		];
+	});
+
+	it("create an instance", () => {
+		expect(pipe).toBeTruthy();
+	});
+
+	it("should sort array if resultDates are different", () => {
+		let res = spyOn(pipe, "transform").and.callThrough();
+		pipe.transform(talks);
+		expect(res).toHaveBeenCalledWith(talks);
+		expect(res).toHaveBeenCalledTimes(1);
```

Comment:
you have to check that the transform returns the expected result

toHaveBeenCalledWith, toHaveBeenCalledTimes in my opinion not about this case

Reply:
done